lib/tilt/rst.rb

```diff
@@ -0,0 +1,18 @@
+require 'tilt/template'
+require 'pandoc-ruby'
+
+module Tilt
+  # Pandoc reStructuredText implementation. See:
+  # http://pandoc.org/
+  # Use PandocTemplate and specify input format
+  class RstTemplate < PandocTemplate
```

Comment:
Maybe call this `RstPandocTemplate` and the file should be `rst-pandoc.rb`? We might want to use `RstTemplate` for the official tool.